_component_design/alert-bar.md

```diff
@@ -3,9 +3,7 @@ title: Alert Bar
 parent: Notifications
 layout: component
 category: Components
-usage: |
-  These tooltips provide users with additional information while completing
-  tasks and alert them to new features.
+usage: Notification bars are best used for information that will remain static barring user intervention. Such as an Info message surrounding a     new feature available to a user, or an Error notification informing the user there credit card has been unable to be processed this month.
```

Comment:
Use notification bars for information that will remain static unless
changed by a user. For example, you could use notification bars for
Info messages about a new feature available to a user, or an Error
message informing the user that their credit card was unable to be
processed this month.